openmdao/visualization/meta_model_viewer/tests/test_view_mm_cmd_line.py

```diff
@@ -26,10 +27,21 @@ def test_invalid_metamodel(self):
         script = os.path.abspath(example.__file__).replace('.pyc', '.py') # PY2
         cmd = 'openmdao view_mm {} -m {}'.format(script, 'interp')
         output = subprocess.check_output(cmd.split()).decode('utf-8', 'ignore')
-        expected_output = '\n'.join([
-            "Metamodel 'interp' not found.",
-            " Try one of the following: ['mm']."
-        ])
+        expected_output = (
+            "\nMetamodel 'interp' not found. Try one of the following:\n"
+            "\nopenmdao view_mm -m mm {}\n".format(script)
+        )
         self.assertTrue(
-            expected_output in output.replace('\r', ''),
+            expected_output in output,
             msg='Metamodel was found when it should not have. Check example.')
+
+    def test_not_metamodel(self):
+        script = os.path.abspath(example.__file__).replace('.pyc', '.py') # PY2
```

Comment:
you can get rid of this `replace` now (here and above) since we no longer support PY2